amida-tech/indaba-client
# Bug/inba 756 jf export ie fix

#### What does this PR do?
Allows data exports for IE as well as Chrome.


#### How should this be manually tested?
In Chrome, confirm that export download for projects continues to successfully download a zip file. Then do the same on IE11. Further updates maybe required to support other browsers in the future. 

#### Background/Context

#### Screenshots (if appropriate):
